packages/webview_flutter/tizen/src/webview_factory.cc

```diff
@@ -24,21 +24,22 @@ WebViewFactory::WebViewFactory(flutter::PluginRegistrar* registrar,
                                flutter::TextureRegistrar* textureRegistrar)
     : PlatformViewFactory(registrar), texture_registrar_(textureRegistrar) {
   char* path = app_get_data_path();
+  std::string path_string;
   if (!path || strlen(path) == 0) {
-    path = "/tmp/";
+    path_string = "/tmp/";
+  } else {
+    path_string = path;
+    free(path);
+    path = nullptr;
   }
-  LOG_DEBUG("application data path : %s\n", path);
-  std::string localstoragePath = path + std::string("StarFish_localStorage.db");
-  std::string cookiePath = path + std::string("StarFish_cookies.db");
-  std::string cachePath = path + std::string("Starfish_cache.db");
+  LOG_DEBUG("application data path : %s\n", path_string.c_str());
+  std::string localstoragePath =
+      path_string + std::string("StarFish_localStorage.db");
```

Comment:
By the way you can rename all the local variables in this file with snake casing.